data/hai/hai reveal 2 start.txt

```diff
@@ -0,0 +1,368 @@
+		"Hai Rescue: Yeertle Family (offer count)" < 3
+		not "Hai Rescue: Yeertle Family: offered"
+	on offer
+		"Hai Rescue: Yeertle Family (offer count)" ++
+		conversation
+			`You look ahead a few dozen meters and you see a short man with a strange necklace. As he walks past a power transfer unit, the man shimmers and you see faintly-visible, squirrel-like, features. It seems the necklace is a projector of a human hologram concealing a Hai. He is sticking to the shadows, trying to avoid being seen.`
+			`	Do you want to talk to him?`
+			choice
+				`	(Talk to him.)`
+				`	(Come back later.)`
+					defer
+			`	The man's expression doesn't change much as you approach. His holoprojector may not be sophisticated enough to translate Hai facial expressions. You can see exposed wires on the device; this is probably a makeshift projector he produced with limited resources. A well-made version might keep Hai hidden in human space for a long time.`
+			`	From the shaking voice, you can tell he's frightened.`
+			`	"C-can I help you?"`
+			choice
+				`	"You're a Hai. Why are you this far into human space?"`
+					goto far
+				`	"Do you need a ride home to Hai space?"`
+					goto "ride offer"
+			label far
+			`	He pauses for a second, perhaps pondering if he can trust you with the answer.`
+			`	"I'm a holographer from Icelake. I chartered a flight from Icelake to Allhome with my family. They would have gone on vacation while I would have been at a conference. Instead, the human merchants locked us in a cell and took us here."`
+			`	"My daughter is very small for her age, and was able to sneak through a vent to unlock our cell. The ship was under-crewed, so we overpowered them, but they had already landed. The ship was locked down; we could not take off. Since then, we've been hiding near here."`
+			`	"Can you take us back to <destination>? I have considerable savings back there, and can pay you well. Say, <payment>?"`
+			choice
+				`	"Sure!"`
+				`	"No, I can't."`
+					decline
+			`	"My name is Duniip Yeertle, by the way. My wife and three children are nearby. They have holoprojectors as well."`
+				goto leave
+			label "ride offer"
+			`	"Yes! Some crooked human merchants locked us in a cell and took us here. I've been unable to find anyone who has even heard of Hai space, and I'm afraid to approach the Syndicate. You're the first lead I have!"`
+			`	"I promise I'll pay you well when we return. I can afford <payment>... when I'm back at <destination>."`
+			`	"My name is Duniip Yeertle, by the way. My wife and three children are nearby. They have holoprojectors as well."`
+			label leave
+			`	"Follow me back to my house, and we will make our preparations."`
+			choice
+				`	(Follow him.)`
+			`	You walk down a corridor to a stairway that leads to a brick wall. Duniip walks through the wall; apparently this is another of his holograms. You pass through and see a triad of energetic Hai children eating dinner with their mother.`
```

Comment:
```suggestion
			`	You walk down a corridor to a stairway that leads to a brick wall. Duniip walks through the wall; apparently this is another of his holograms. You pass through and see a trio of energetic Hai children eating dinner with their mother.`
```
I think trio works better here based on [this](https://comparewords.com/triad/trio).